package.json

```diff
@@ -58,6 +72,7 @@
     "@typescript-eslint/parser": "^5.56.0",
     "@vscode/l10n-dev": "^0.0.26",
     "@vscode/test-electron": "^2.3.0",
+    "esbuild": "^0.18.6",
```

Comment:
Opps, this is not needed just yet as this is what does the bundling. Will remove until next PR.